test/bigquery.ts

```diff
@@ -170,6 +170,14 @@ describe('BigQuery', () => {
   });
 
   describe('instantiation', () => {
+    function setHost(host: string) {
```

Comment:
In other test suites, we'll do a before/after to cache/restore "process.env.BIGQUERY_EMULATOR_HOST" to any value that may have been set by the user before running the tests.